catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -490,24 +497,28 @@ private static void setMetacardEmailAttribute(List<EmailAddressType> emailAddres
 
     private static void setMetacardPhoneNumberAttribute(List<TelephoneNumberType> phoneNumbers,
             String metacardAttribute, MetacardImpl metacard) {
-        List<String> metacardPhoneNumbers = new ArrayList<>();
 
-        for (TelephoneNumberType digits : phoneNumbers) {
+        List<String> metcardPhoneNumbers = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(phoneNumbers)) {
+            for (TelephoneNumberType digits : phoneNumbers) {
+                metcardPhoneNumbers.add(getPhoneNumber(digits));
 
-            if (StringUtils.isBlank(digits.getExtension())) {
-                metacardPhoneNumbers.add(String.format("(%s) %s",
-                        digits.getAreaCode(),
-                        digits.getNumber()));
-            } else {
-                metacardPhoneNumbers.add(String.format("(%s) %s extension %s",
-                        digits.getAreaCode(),
-                        digits.getNumber(),
-                        digits.getExtension()));
+                if (CollectionUtils.isNotEmpty(metcardPhoneNumbers)) {
+                    metacard.setAttribute(metacardAttribute, (Serializable) metcardPhoneNumbers);
+                }
             }
         }
-        if (CollectionUtils.isNotEmpty(metacardPhoneNumbers)) {
-            metacard.setAttribute(metacardAttribute, (Serializable) metacardPhoneNumbers);
+    }
+
```

Comment:
Would you be able to use StringUtils.joinWith() instead of this? https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#joinWith(java.lang.String,%20java.lang.Object...)